### ISSUE TITLE
Add a standard way of presenting references to elements of the standard

### ISSUE THREAD

**Issue by robredpath** [IATI/design-system#29]
E.g. the dashboard has this:

<img width="184" alt="Screenshot 2024-11-29 at 09 33 38" src="https://github.com/user-attachments/assets/561f7f3f-7f1f-4d7d-9c95-93f7e25a6af4">

and the website has this:

<img width="264" alt="Screenshot 2024-11-29 at 09 35 14" src="https://github.com/user-attachments/assets/2d29239a-058f-4fe6-ac9f-154871ddb41e">

**Comment by tillywoodfield** [comment]
Agreed to use font [Roboto Mono](https://fonts.google.com/specimen/Roboto+Mono)

**Comment by robredpath** [comment]
@tillywoodfield and I discussed this last week - although we were keen to maintain a "computer-ey-feeling" font (and therefore have chosen a monospace font) we also want to make sure that we don't make this design feel like what the link leads to isn't for people who work in spreadsheets. I think we can do that through the choice of colours - and to that end I'm tending towards the styling already used on iatistandard.org

**Comment by codemacabre** [comment]
I've had to darken the blue used as it didn't pass colour contrast against the white background. This is my current suggestion for inline code, block-level code, and references to elements in the standard:

![Image](https://github.com/user-attachments/assets/615eb662-847b-4fdf-9b57-3f32cf570aa1)
